moon-store-mac-os/moon-store-mac-os/Domain/Models/ProductModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ProductModel.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/23/24.
+//
+
+import SwiftUI
+
+enum ProductCategory: String, Decodable {
+    case laptop
+    case phone
+    case audio
+    case pc
+    case printer
+    case battery
+    case wearable
+    case home
+    case peripheral
+    case networking
+    
+    var title: String { rawValue.capitalized }
+    
+    var color: Color {
+        switch self {
+            case .laptop, .phone, .pc: .msPrimary
+            case .audio, .printer, .battery: .msOrange
+            case .wearable, .home, .peripheral, .networking: .msGreen
```

Comment:
Necesitas agregar una opcion unicamente para el `@unknown` porque si esto depende del backend, no siempre seran los mismos.

Reply:
https://www.avanderlee.com/swift/unknown-default-enums-in-swift/